assets/css/edd-admin.css

```diff
@@ -57,24 +57,28 @@ a.edd-delete:hover {
 	z-index: 1;
 }
 
-.edd-from-to-wrapper input:focus {
-	z-index: 2;
-	outline: auto 5px -webkit-focus-ring-color;
-	border-color: #eee;
-	box-shadow: none;
-}
-
 .edd-from-to-wrapper input[name="start"],
 .edd-from-to-wrapper input[name="start-date"],
 .edd-from-to-wrapper input[name="filter_from"] {
-	border-right-color: #eee;
+	border-radius: 4px 0 0 4px;
```

Comment:
@LisaCee For the border radius changes let's only _remove_ the round corners where needed and leave the other radi as-is. That way if core changes to a `3px` or `5px` radius we don't need to update.